js/apps/admin-ui/src/components/users/UserDataTable.tsx

```diff
@@ -112,6 +115,36 @@ export function UserDataTable() {
   const [key, setKey] = useState(0);
   const refresh = () => setKey(key + 1);
 
+  const handleChange = async (
+    checked: boolean,
+    event: React.FormEvent<HTMLInputElement>,
+  ) => {
+    try {
+      await adminClient.users.update(
+        { id: event.currentTarget.id },
+        toUserRepresentation({
+          enabled: checked,
+        }),
+      );
+      refresh();
+    } catch (error) {
+      console.log(error);
+      refresh();
+    }
+  };
+
+  // eslint-disable-next-line react/no-unstable-nested-components
+  const ToggleEnabled = (user: UserRepresentation) => {
+    return (
+      <Switch
+        id={user.id}
+        aria-label="User when enabled"
```

Comment:
This needs to be localized like other labels. I think we should actually wrap this in a tooltip.